### DISCUSSION TITLE
Predictive Electric Vehicle Cooling (Project 194)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#25] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #194](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Predictive%20Electric%20Vehicle%20Cooling)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by Suyash7j** [comment]
Hi, my name is Suyash. I am interested in working on this exciting project. Please reach out to me if you want to collaborate.

**Reply by Arun-Mohapatra** [reply]
Hello, I am Arun, a student at NIT Rourkela, India. I am interested in working and collaborating on this project.

**Comment by mukundvats** [comment]
hello i am working on tis project project number 194 please reach me out to collabrate

**Reply by Arun-Mohapatra** [reply]
Hello 
I am Arun, and I am interested to collaborate with you on this. 
Please reach out to me via mail- arunmohapatra987@gmail.com.

**Comment by VineetRathod** [comment]
Hi, 
I am Vineet Rathod a student at Coventry University and I am interested in working and collaborating on this project.

**Comment by mtallas** [comment]
Hello I am Murat, a student at Polytechnic University of Tirana, Albania. I am interested in working and collaborating on this project.

**Comment by Gokarna-Bahadur-Thapa** [comment]
Could you please provide information about the current application deadline and validity period for this project? Thank you.

**Reply by robertogl** [reply]
Hi @Gokarna-Bahadur-Thapa,

This project makes you eligible to participate in the [Sustainability and Renewable Energy Challenge](https://www.mathworks.com/academia/student-challenge/sustainability-and-renewable-energy-challenge-2023.html). If you want to participate in the challenge you will have to submit a solution by January 5, 2024. Otherwise, you can ask for more time if you need it. Regarding the project, there is no plan to archive it yet.

Please let me know if you need anything else.

**Comment by atharvaajaynikam4601-netizen** [comment]
Hi all — I'm Atharva, a Mechatronics Master's student at Hochschule Ravensburg-Weingarten, Germany, working solo on Predictive Electric Vehicle Cooling (#194).

The short version: reactive thermostat cooling reacts too late — by the time it turns on, the battery's already hot. I built a Simscape Fluids cooling plant (cold plate, pump, radiator) and put three controllers on it — a reactive baseline, a lookahead-predictive controller, and a full nonlinear MPC — then benchmarked them against each other on a UDDS drive cycle and a nastier fast-charge stress scenario. The predictive/MPC controllers eliminate every safety-threshold excursion the reactive one hits, at the cost of spending cooling energy earlier instead of saving it — a real trade-off, not a free win.

Repo's here if you want to poke at it or compare notes: https://github.com/atharvaajaynikam4601-netizen/Predictive-Electric-Vehicle-Cooling

Would genuinely enjoy hearing from anyone else tackling this one — especially if you found a different angle on the predictive controller.
